### ISSUE TITLE
Use CSS-in-JS from @patternfly/react-styles

### ISSUE THREAD

**Issue by jennydaman** [FNNDSC/ChRIS_store_ui#167]
https://www.npmjs.com/package/@patternfly/react-styles

Continues https://github.com/FNNDSC/ChRIS_store_ui/pull/162

**Comment by somya51p** [comment]
@jennydaman Can I look into this one too. Can you help me out finding the resources for [@pattern/react-styles](https://www.npmjs.com/package/@patternfly/react-styles)

**Comment by DivyanshiSingh** [comment]
@jennydaman I would like to take this up next.

**Comment by jennydaman** [comment]
@somya51p as mentioned in our conversation in #162, I am not sure where to find documentation on @patternfly/react-styles either. It'll probably take a skim of the source code and a good IDE.

@DivyanshiSingh this issue depends on #162 which has not been merged yet.

**Comment by Akpjunior94** [comment]
Hi @mairin, My Name is Akpan Abraham, I'm an Outreachy intern and want to contribute to the project "Create external app registration system for React-based user interface". Can I be assigned to this issue please?

**Comment by jennydaman** [comment]
Hey everyone sorry for missing these replies!

The issue is about using the "CSS-in-JS" syntax so that CSS files are imported as JavaScript objects, instead of referring to CSS classes by name using strings. For example, instead of


```jsc
import 'styles.css';
...
<div className="something" />
```


We prefer

```jsx
import { styles } from 'styles.css';
...
<div className={styles.something} />
```

The advantage of CSS modules are, class names can be scoped and they can also be checked to exist at build time.

**Comment by mairin** [comment]
This issue is still open, see https://github.com/FNNDSC/ChRIS_store_ui/issues/167#issuecomment-957889957 for details on how to complete this.

**Comment by somya51p** [comment]
> This issue is still open, see [#167 (comment)](https://github.com/FNNDSC/ChRIS_store_ui/issues/167#issuecomment-957889957) for details on how to complete this.

@mairin I would like to work on this issue. Please assign me the same.
ThankYou!!

**Comment by mairin** [comment]
@somya51p Sure I just assigned to you.

**Comment by Mfonn** [comment]
@jennydaman I will like to work on this issue.

**Comment by somya51p** [comment]
Hey @Mfonn, I am already working on it :)

**Comment by Mfonn** [comment]
Okay then, I'll find another issue to work on.

**Comment by somya51p** [comment]
Hey @jennydaman, I have created the script to convert CSS to CSS modules, in which form shall I create the PR for review.
